parsing/src/parts.ts

```diff
@@ -260,17 +285,14 @@ function getRawResources(data: any[]): { [key: string]: RawResource } {
         "Stone": 69900,
         "Sulfur": 10800,
         "Water": 9007199254740991,
-    }
+    };
 
     data
         .filter((entry: any) => entry.NativeClass === "/Script/CoreUObject.Class'/Script/FactoryGame.FGResourceDescriptor'")
         .flatMap((entry: any) => entry.Classes)
         .forEach((resource: any) => {
-            const className = resource.ClassName
-                .replace('Desc_', '')
-                .replace(/_C$/, '')
-                .toString();
-            const displayName = resource.mDisplayName;
+            const className: string = getPartName(resource.ClassName);
```

Comment:
```suggestion
            const className = getPartName(resource.ClassName);
```

if `getPartName` has a return type of string, re-defining the type in the var is not required. If it doesn't have a return type, it needs one.